Backend/src/swagger.json

```diff
@@ -1286,6 +1286,35 @@
         }
       }
     },
+    "/users/{id}": {
+      "get": {
+        "summary": "Get user by id",
+        "tags": [
+          "User"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returned the user requested sucessfully!"
+          },
+          "400": {
```

Comment:
According to the implementation, it's never sent 400 error.

Reply:
Yeap, you are right i will have to revise all my documentations.